### ISSUE TITLE
Can't use custom base64 font in lottie animation loaded in Android Webview

### ISSUE THREAD

**Issue by rajatsangrame** [airbnb/lottie-web#2942]
In the HTML file, I have 2 elements, one is a sample text and the second is lottie-container. I am trying to load 
`Poppins-Bold.otf` as a base64 string and it works fine for the animation. 

   ```
 function loadData(base64font, jsonData) {
         
            let url = "url(data:font/truetype;charset=utf-8;base64," + base64font + ") format('opentype')";
            console.log('url', typeof (url))
            let font = new FontFace('Poppins-Bold', url);
            font.load().then(function (loadedFace) {

                console.log('tag', document.fonts.check("14px Poppins-Bold")); //true
                console.log('tag', loadedFace.status); // loaded            
                document.fonts.add(loadedFace);
                loadAnimation(jsonData);
            }).catch(function (error) {
                console.log(error);
            });
}

  function changeText(text) {
            textData.f = 'Poppins-Bold';
            textData.t = 'Domestic LPG price hiked by Rs 50 per 14.2-kg cylinder, first hike since July 2022';
            animation.renderer.elements[0].updateDocumentData(textData, 0);
 }
```

<img width="372" alt="Screenshot 2023-03-10 at 6 08 47 PM" src="https://user-images.githubusercontent.com/33744237/224318522-8ecc28b7-f484-437b-9856-1446487c788d.png">

But when I load the HTML inside an Android WebView, the sample text is updated with the new font but not working for Lottie Animation.

The console logs are also correct in android webview.
```
console.log('tag', document.fonts.check("14px Poppins-Bold")); //true
console.log('tag', loadedFace.status); // loaded          
```

<img width="230" alt="Screenshot 2023-03-10 at 6 19 55 PM" src="https://user-images.githubusercontent.com/33744237/224321120-4b07dc3f-cb17-4f30-8766-42773a184c02.png">

**Comment by rajatsangrame** [comment]
@bodymovin @gpeal Can you please suggest something on the issue?

**Comment by gpeal** [comment]
I think you meant to post this on https://github.com/airbnb/lottie-web/

**Comment by rajatinrizzle** [comment]
@gpeal Didn't get you. this is posted in the [https://github.com/airbnb/lottie-web/](https://github.com/airbnb/lottie-web/)  issues only.

**Comment by gpeal** [comment]
@rajatinrizzle Yes but if you're playing Lottie in a WebView then it's lottie-web.

**Comment by rajatsangrame** [comment]
@gpeal I think you still misunderstood, I am saying this issue is part of [lottie-web](https://github.com/airbnb/lottie-web/) only, not [lottie-android](https://github.com/airbnb/lottie-android)

**Comment by gpeal** [comment]
@rajatinrizzle Ah yes. I don't normally get tagged here so I assumed this was lottie-android without looking.

**Comment by bodymovin** [comment]
Hi, unfortunately I don't have an easy way to test Android WebViews, have you tested directly providing the font to the animation instead of loading it afterwards to check if it works?  
Another test would be to see if loading the font via css behaves differently.

**Comment by rajatsangrame** [comment]
@bodymovin 
1. Let me provide you sample android project so you can easily test Android WebViews.
2. I didn't get how can I provide the font directly to the animation. If you meant `"fPath": "https://fonts.googleapis.com/css2?family=Poppins"` in the Lottie JSON  it didn't work for me.
3. I have tried loading font via CSS. It is working but there are character spacing issues in Android WebView. On the web, it is working fine. Check the image for reference. 
I chose loading font with base64 string because it is easy to inject in the Android Webivew and with font-face API I get a `promise` that make sure the font is loaded.
```
<style>
    @import url('https://fonts.googleapis.com/css2?family=Sono:wght@800&display=swap'); */
</style>
```
<img width="343" alt="mobile" src="https://user-images.githubusercontent.com/33744237/224639335-69333d6f-5f4e-4bad-a31b-f0071321c6fe.png">
<img width="394" alt="new" src="https://user-images.githubusercontent.com/33744237/224640734-618e90d4-6516-4419-86c4-ab84d0c82934.png">

**Comment by rajatsangrame** [comment]
@bodymovin  Can you please check this [sample project ](https://github.com/rajatinrizzle/test-lottie)to Android Webview

**Comment by rajatsangrame** [comment]
@bodymovin Can you please provide some suggestions to debug this issue? Can you navigate me to the code in `lottie.js` where the browser is using the default font when the font is loaded in DOM?
